sample/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
 dependencies {
     compile 'com.android.support:appcompat-v7:22.1.1'
     compile 'com.android.support:recyclerview-v7:22.1.1'
-    //compile project(':library')
-    compile 'com.prolificinteractive:material-calendarview:0.3.2'
+    compile project(':library')
+    //compile 'com.prolificinteractive:material-calendarview:0.3.2'
```

Comment:
Can you add a comment about why it's here so it's not confusing for anyone in that case?